pyhocon/config_parser.py

```diff
@@ -239,6 +239,7 @@ class ConfigParser(object):
         '\\#': '#',
         '\\!': '!',
         '\\"': '"',
+        '\/': '/',
```

Comment:
Should it be `\\/` instead? Do you have an example where you use it?

Reply:
As shown in #293 , json use '\/' in unix path string like {"path":"\/var"}.

pyhocon does not parse it correctly as jq does.

```bash
# echo '{"path":"\/var"}' | jq .
{
  "path": "/var"
}
# echo '{"path":"\/var"}' | pyhocon
{
  "path": "\\/var"
}
```

Reply:
This syntax is described in [RFC 8259 Section 7 Strings](https://www.rfc-editor.org/rfc/rfc8259.html#section-7)